REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.5
-OpenCL
+OpenCL 0.5.2
```

Comment:
I want people to update. it should be strictly better and if not I want to find out!